### DISCUSSION TITLE
Switching between computers using a KVM switch breaks connection

### DISCUSSION THREAD

**Discussion by YodAmygdala** [apohl79/audiogridder#1270] [Help] [answered]
Hi! 

Not sure if this problem I'm having is with AudioGridder or my KVM switch, but I'll start here.

I running a Macbook Pro 2018 with macOS 12.6.3 with Cubase 12, Logic 10.6.3 (and Ableton Live 12) for the plugin and a PC with Win10 64 as a server. AG version is 1.2.0. I'm using both computers on the same screen, keyboard and mouse using this KVM: https://www.direktronik.se/direktronik/natverk/kvm-switcharextenders/kvm-switchar-for-kontor/2-port-hdmi-4k-kvm-switch-4xusb2/?variationCode=20109978 

The computers are connected with ethernet and this connection works fine as long as the server is up on my screen, but as when I switch the KVM to my Macbook the connection is broken. When I try to switch back to the PC I can't get an image before I close the DAW on my Macbook (before switching to PC). This happens in both Cubase and Logic. 

I'm not very proficient in servers, networks and such so maybe there's an easy solution here or something, so please enlighten me. 

HALP! :)

Thank you!

**Comment by apohl79** [comment, marked as answer]
The problem is, that when you switch to the mac, you are disconnecting the display from windows. This breaks AG as it can't render the plugin UI anymore. You always need a display connected or you use VNC for example.
